content/ro/docs/what-is-opentelemetry.md

```diff
@@ -0,0 +1,153 @@
+  denumire pentru tipurile comune de date de telemetrie
+- API-uri care definesc modul de generare a datelor de telemetrie
+- [SDK-uri lingvistice](../languages) care implementează specificațiile,
+  API-urile și exportul datelor de telemetrie
+- Un [ecosistem de biblioteci](/ecosistem/registry) care implementează
+  instrumentație pentru biblioteci și framework-uri comune
+- Componente de instrumentație automată care generează date de telemetrie fără a
+  necesita modificări de cod
+- [Colectorul OpenTelemetry](../collector), un proxy care primește, procesează
+  și exportă date de telemetrie
+- Diverse alte instrumente, cum ar fi
+  [Operatorul OpenTelemetry pentru Kubernetes](../platforms/kubernetes/operator/),
+  [Diagramele OpenTelemetry Helm](../platforms/kubernetes/helm/) și
+  [resursele comunității pentru FaaS](../platforms/faas/)
+
+OpenTelemetry este utilizat de o gamă largă de
+[biblioteci, servicii și aplicații](/ecosystem/integrations/) care au
+OpenTelemetry integrat pentru a oferi observabilitate în mod implicit.
+
+OpenTelemetry este susținut de numeroși [furnizori](/ecosystem/vendors/), mulți
+dintre ei oferind suport comercial pentru OpenTelemetry și contribuind direct la
+proiect.
+
+## Extensibilitate
+
+OpenTelemetry este conceput pentru a fi extensibil. Câteva exemple despre cum
+poate fi extins includ:
+
+- Adăugarea unui receptor la OpenTelemetry Collector pentru a accepta date de
+  telemetrie dintr-o sursă personalizată
+- Încărcarea bibliotecilor de instrumentație personalizate într-un SDK
+- Crearea unei [distribuții](../concepts/distributions/) a unui SDK sau a
+  Colectorului adaptată unui caz de utilizare specific
+- Crearea unui nou exportator pentru un backend personalizat care nu acceptă
+  încă protocolul OpenTelemetry (OTLP)
+- Crearea unui propagator personalizat pentru un format de propagare a
+  contextului nestandard
+
+Deși majoritatea utilizatorilor s-ar putea să nu aibă nevoie să extindă
+OpenTelemetry, proiectul este conceput pentru a face acest lucru posibil la
+aproape orice nivel.
+
+## Istorie
+
+OpenTelemetry este un proiect al [Cloud Native Computing Foundation][] (CNCF)
+care este rezultatul unei [fuziuni] între două proiecte anterioare,
+[OpenTracing](https://opentracing.io) și [OpenCensus](https://opencensus.io).
+Ambele proiecte au fost create pentru a rezolva aceeași problemă: lipsa unui
+standard pentru instrumentarea codului și trimiterea datelor de telemetrie către
+un backend Observability. Întrucât niciunul dintre proiecte nu a reușit să
+rezolve problema independent, ele au fuzionat pentru a forma OpenTelemetry și
+a-și combina punctele forte, oferind în același timp o soluție unică.
+
+Dacă utilizezi în prezent OpenTracing sau OpenCensus, poți afla cum să migrezi
+la OpenTelemetry în [Ghidul de migrare](../migration/).
+
+[fuziuni]:
+  https://www.cncf.io/blog/2019/05/21/a-brief-history-of-opentelemetry-so-far/
+
+## Ce urmează?
+
+- [Începe](../getting-started/) &mdash; dă-i drumul!
```

Comment:
nit: Perhaps more like: `Să începem - treci direct la acțiune!`?  - my concern with `dă-i drumul` is that it can also mean "go away" in other contexts